### PR TITLE
[#9]: Configurable operators

### PR DESCRIPTION
**Pull request by nickdeis**
For ticket #9 , a very naive implementation, but it should work given that react-structured filter is more about generating a tree and not actually implementing the filter.

### CODE REVIEW

**Review by joshcarr — CHANGES_REQUESTED** [review]
Can you address the eslint complaints? Run `npm run lint` to see the output.

**Comment by nickdeis** [comment]
Sure thing.
EDIT: Done

File: src/tokenizer/index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import Token from './token';
 import KeyEvent from '../keyevent';
 import Typeahead from '../typeahead';
 import classNames from 'classnames';
+import objectAssign from "object-assign";
```
Comment (joshcarr) [inline src/tokenizer/index.js]: You are importing this, but I don't see it in `package.json`. Any reason why?

Comment (nickdeis) [reply src/tokenizer/index.js]: It's a dependency for React. But maybe it should be added? It's better to be declarative.

Comment (joshcarr) [reply src/tokenizer/index.js]: Yes please do explicitly add it to `package.json`

File: package.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
     "fuzzy": "^0.1.0",
     "moment": "*",
     "react-datetime": "^2.1.0",
-    "react-onclickoutside": "^4.9.0"
+    "react-onclickoutside": "^4.9.0",
+    "object-assign":"^3.0.0"
```
Comment (nickdeis) [inline package.json:18]: This is the version used by react-datetime

**Comment by nickdeis** [comment]
Updated package.json

**Comment by joshcarr** [comment]
Released in [v4.4.0](https://github.com/joshcarr/react-structured-filter/releases/tag/v4.4.0)

**Pull request merged**